updatehub/src/main.rs

```diff
@@ -25,6 +25,10 @@ enum EntryPoints {
 struct ClientOptions {
     #[argh(subcommand)]
     commands: ClientCommands,
+
+    /// change the client socket to listen
+    #[argh(option)]
+    listen_socket: Option<String>,
```

Comment:
You could make use of the `default` flag on `argh` so we don't need to handle it in code.